libs/frontend/feature-render-builder/src/components/ProseMirrorEditor.ts

```diff
@@ -201,12 +205,51 @@ export const ProseMirrorEditor = defineComponent({
       },
     )
 
+    const cursorLinkAttrs = ref()
+    let linkElement: Element | undefined = undefined
+
+    watch(editViewTxCount, () => {
+      const { editView } = editor.value
```

Comment:
This might end up being inefficient, I wonder if we can specifically observe whether the selection changed, by setting a different variable in `dispatchTransaction`. I'll add a note to check it eventually!